backend/pkg/modules/internal/fuzzer/fuzzer.go

```diff
@@ -132,6 +132,8 @@ func (p *pluginFuzzer) sendAPIFindingsNotification(ctx context.Context, apiID ui
 		return fmt.Errorf("failed to create 'APIFindings' notification, err=(%v)", err)
 	}
 
+	b, err := json.Marshal(notification)
+	logging.Logf("[Fuzzer] sendAPIFindingsNotification(%v): notification=%v", apiID, string(b))
```

Comment:
let's do it only if log.level is debug - not sure we want to marshal all the time

Reply:
it's only for integration purpose, the trace will disappear after. But I know that apifindingsnotification already works, then I will remove it.